.github/workflows/release.yml

```
@@ -0,0 +1,202 @@
+      version_type:
+        description: 'Version increment type'
+        required: true
+        type: choice
+        options:
+        - patch
+        - minor
+        - major
+        default: patch
+  push:
+    tags:
+      - 'v*'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)